doc/nrf/gs_testing.rst

```diff
@@ -120,3 +120,74 @@ To connect to the nRF9160-based kit with LTE Link Monitor, perform the following
    .. note::
 
       In the case of nRF9160 DK, the reset button must be pressed to restart the device and to start the application.
+
+.. _gs_debugging:
+
+Debugging an application
+************************
+
+To debug an application, set up the debug session as described in `Debugging an application`_ in the |nRFVSC| documentation.
+
+If you use a multi-core SoC, for example from the nRF53 Series, you need a single debug session to debug the application core firmware.
```

Comment:
Does SoC need to be opened as System on Chip? Or do we consider the abbreviation common enough?

Reply:
I think it's common enough. We also have it in the glossary.